core/src/test/resources/config-dev.yaml

```diff
@@ -5,6 +5,10 @@ streamValidatorConfig:
   properties:
     validator.invalid-id: 13
 
+schemaRegistrarConfig:
+  className: com.homeaway.streamingplatform.extensions.validator.SchemaRegistrarIT$ValidSchemaRegistrar
```

Comment:
Once you have the real implemention of submodule `schema-registry-validator`, can you please replace the above class with the real class.